include/wintls/detail/sspi_handshake.hpp

```diff
@@ -74,25 +115,67 @@ class sspi_handshake {
     }();
 
     auto server_cert = context_.server_cert();
-    if (handshake_type_ == handshake_type::server && server_cert != nullptr) {
-      creds.cCreds = 1;
-      creds.paCred = &server_cert;
-    }
+    bool is_tlsv13 = !!(static_cast<int>(context_.method_) & static_cast<int>(method::tlsv13));
```

Comment:
I'm not really a fan of this C type code as it depends on the underlying value of the enum and is a bit hard to read. Please consider using an inline lambda similar to what is being done for the `usage` variable above.